### DISCUSSION TITLE
How to get started with code?

### DISCUSSION THREAD

**Discussion by Duq42** [pybricks/support#1501] [Q&A]
Hi all,
I've installed PyBricks on a Technic hub and it's connected.
Now I want to try and see how it works using just code, but I can't get rid of the blocks.
This page: https://pybricks.com/install/running-programs/ says "just copy and paste this snippet" but there's nowhere to paste.
What am I doing wrong?
Thanks!
Ronald

**Comment by laurensvalk** [comment]
When you create a new project, you can choose between blocks and Python.

It sounds like you'd like to use Python:

![image](https://github.com/pybricks/support/assets/12326241/08980da5-edf1-41f4-b63d-347b975e88f8)

**Comment by Duq42** [comment]
Yes, that's the step I missed.
